### DISCUSSION TITLE
Efflux Process

### DISCUSSION THREAD

**Discussion by pburkat** [Open-Systems-Pharmacology/Forum#959] [Questions & Problems]
Greetings, where might I find information on efflux processes for compounds in the brain compartment?

**Comment by StephanSchaller** [comment]
Can you specify?

If you add an efflux transporter in PK-Sim and the respective transport kinetics in the ADME Tab of your compound, you can look at the full formula only in MoBi (--> export model to MoBi and then check the Molecules Building Block and then the compound for which you have defined the efflux, there should be the effluxes listed).

**Reply by pburkat** [reply]
I'm interested in morphine and it's glucuronide transport across the blood barrier via efflux transporter P-glycoprotein (ATP-binding cassette, subfamily B, member 1 (ABCB1).

**Comment by StephanSchaller** [comment]
You could load the expression profile for P-gp (search for ABCB1 in the gene expression database)

**Reply by pburkat** [reply]
Thanks for the help.
